inc/saiacl.h

```diff
@@ -1660,10 +1678,43 @@ typedef enum _sai_acl_entry_attr_t
      */
     SAI_ACL_ENTRY_ATTR_FIELD_IPV6_NEXT_HEADER,
 
+    /**
+     * @brief Bridge Type (field mask is not needed)
+     *
+     * This is to match the bridge type of the packet.
+     * This would be useful when the ACL table/group is
+     * bound to a switch bind point and the ACL needs to
+     * be applied only on bridges of type 802.ID or only
+     * on the 802.1Q bridge.The value for this match will
+     * the enum values of sai_bridge_type_t.
+     * i.e. SAI_BRIDGE_TYPE_1Q or SAI_BRIDGE_TYPE_1D.
+     *
+     * @type sai_acl_field_data_t sai_int32_t
+     * @flags CREATE_AND_SET
+     * @default disabled
+     */
+    SAI_ACL_ENTRY_ATTR_FIELD_BRIDGE_TYPE,
+
+    /**
+     * @brief Bridge ID (mask is not needed)
+     *
+     * This is to match the packet's Bridge ID.
+     * This will be useful when the ACL table/group
+     * is bound to a switch bind point and the rules need
+     * to identify the Bridge on which the packet is
+     * present.
+     *
+     * @type sai_acl_field_data_t sai_object_id_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_BRIDGE
+     * @default disabled
```

Comment:
I guess _@default_ here should be _SAI_NULL_OBJECT_ID_.

Reply:
Same reason as the above. The similar example is SAI_ACL_ENTRY_ATTR_FIELD_IN_PORT where the default state  for sai_acl_field_data_t  is disabled. I guess this is convention followed in the existing saiacl.h header.
@kcudnik Is the understanding correct ?